addon/poll-task.js

```diff
@@ -1,6 +1,7 @@
 import Ember from 'ember';
 import { run } from '@ember/runloop';
 import { assert } from '@ember/debug';
+import { settled } from '@ember/test-helpers';
```

Comment:
This will fail in non-test code. We should refactor things a bit so that the test related code actually comes from addon-test-support instead of addon/

Reply:
Agree, which is why I pinged you yesterday about this :)